.storybook/preview.js

```diff
@@ -27,3 +28,24 @@ for (let i = -3; i <= 6; i++) {
 }
 addDecorator(withPaddings);
 addParameters({ paddings });
+
+// Create viewports using widths defined in design tokens
+const breakpointViewports = Object.keys(breakpoints).map((name) => {
+  return {
+    name,
+    styles: {
+      width: breakpoints[name],
+      height: '100%',
+    },
+    type: 'other',
+  };
+});
+/**
+ * The default viewport (which is not overridden here) is Responsive: 100% x 100%.
+ * After a breakpoint is chosen, the "Reset Viewport" option will use this default.
+ */
```

Comment:
I understand what this is saying now, but I still don't understand its relevance to the code below. It seems like it's just describing a feature of the add-on. Is there something about the "Reset Viewport" feature that impacted how you wrote lines 47–51 that I'm missing?

Reply:
@tylersticka, I'm just trying to explain that the default viewport (aka, `defaultViewport` in the config, which is an option) is not something we're setting, and that it's both the default on page load and the viewport that is used when you use "Reset viewport" (which shows up after you've selected one of the breakpoint-based viewport options).

I tried overriding the default at one point, but decided it was best to leave it as is, and this option wasn't clear to me until I read the addon's docs. If you think this comment isn't helpful, should I just remove it?

Reply:
I've just removed this comment.